dracut.conf.d/fedora.conf.example

```diff
@@ -22,7 +22,7 @@ dbusconfdir=/etc/dbus-1
 dbusinterfacesconfdir=/etc/dbus-1/interfaces
 dbusservicesconfdir=/etc/dbus-1/services
 dbussessionconfdir=/etc/dbus-1/session.d
-dbussystem=confdir/etc/dbus-1/system.d
+dbussystemconfdir/etc/dbus-1/system.d
```

Comment:
I doubt this is what you wanted.

Reply:
Fixed along with another thing I noticed that I had missed.